### DISCUSSION TITLE
1-Click Quick Launch Toolbar

### DISCUSSION THREAD

**Discussion by cheeseandjamsandwich** [valinet/ExplorerPatcher#3774] [Ideas]
The Quick Launch Toolbar is one of the reasons EP is so good to have!  I can't live without it!

But it's a little bit drawn out to add it... 

Right-Click
Toolbars
New Toolbar
C: Drive
Users
Username
AppData*
Roaming
Microsoft
Internet Explorer
Quick Launch
Right-Click
Lock all toolbars
Right-Click
Show Text
Show title
Drag to left
Drag divider

* Assuming you already have Hidden Files showing! lol.

Bit of a faff...

It would be lovely if it was just one of the options in EP's Taskbar Properties!
1-Click, and it's there...

**Comment by Gaafgeel** [comment]
Thank you @cheeseandjamsandwich for your installation manual, I was totally forgotten how hard it was to set it back again after installing the new version 😅
